packages/@uppy/components/src/hooks/remote-source.ts

```diff
@@ -6,11 +6,19 @@ import type {
   UppyEventMap,
 } from '@uppy/core'
 import type { ProviderViews } from '@uppy/provider-views'
-import type { AvailablePluginsKeys } from '@uppy/remote-sources'
 import { dequal } from 'dequal/lite'
 import { Subscribers } from './utils.js'
 
-export type { AvailablePluginsKeys as RemoteSourceKeys }
+export type RemoteSourceKeys =
```

Comment:
isn't this already defined in `@uppy/remote-sources`? we should remove it from there, so we don't have two of the same type